zeppelin/src/main/scala/org/apache/zeppelin/crossdata/CrossdataInterpreter.scala

```diff
@@ -15,40 +15,43 @@
  */
 package org.apache.zeppelin.crossdata
 
-import com.stratio.crossdata.common.result.ErrorSQLResult
-import com.stratio.crossdata.common.result.SQLResult
-import com.stratio.crossdata.common.result.SuccessfulSQLResult
-import com.stratio.crossdata.driver.JavaDriver
-import com.stratio.crossdata.driver.config.DriverConf
-import org.apache.spark.sql.Row
-import org.apache.spark.sql.types.StructType
-import org.apache.zeppelin.interpreter.Interpreter
-import org.apache.zeppelin.interpreter.InterpreterContext
-import org.apache.zeppelin.interpreter.InterpreterResult
-import org.apache.zeppelin.scheduler.Scheduler
-import org.apache.zeppelin.scheduler.SchedulerFactory
-
-import scala.concurrent.duration.Duration
-import scala.concurrent.duration.FiniteDuration
 import java.util.Properties
 import java.util.concurrent.TimeUnit
 
+import com.stratio.crossdata.common.result.{ErrorSQLResult, SQLResult, SuccessfulSQLResult}
+import com.stratio.crossdata.driver.Driver
+import com.stratio.crossdata.driver.config.DriverConf
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types.StructType
 import org.apache.zeppelin.interpreter.Interpreter.FormType
+import org.apache.zeppelin.interpreter.{Interpreter, InterpreterContext, InterpreterResult}
+import org.apache.zeppelin.scheduler.{Scheduler, SchedulerFactory}
 
 import scala.collection.JavaConversions._
+import scala.concurrent.duration.{Duration, FiniteDuration}
 
 object CrossdataInterpreter {
   private val CROSSDATA_SEEDS_PROPERTY: String = "crossdata.seeds"
+  private val CROSSDATA_HTTP_CONNECTION: String = "crossdata.http"
   private val CROSSDATA_DEFAULT_LIMIT: String = "crossdata.defaultLimit"
   private val CROSSDATA_TIMEOUT_SEC: String = "crossdata.timeoutSeconds"
 }
 
 class CrossdataInterpreter(property: Properties) extends Interpreter(property) {
-  private var driver: JavaDriver = _
+  private var driver: Driver = _
 
   def open() = {
     val seeds: java.util.List[String] = (getProperty(CrossdataInterpreter.CROSSDATA_SEEDS_PROPERTY).split(",")).toList
-    driver = new JavaDriver(new DriverConf().setClusterContactPoint(seeds))
+    val driverConf = new DriverConf
+
+    driver = if(getProperty(CrossdataInterpreter.CROSSDATA_HTTP_CONNECTION).toBoolean){
+      val httpHostAndPort = seeds(0).split(":") match { case Array(a, b) => (a, b) }
```

Comment:
`seeds(0).split(":") match { case Array(host, port) =>    driverConf.setHttpHostAndPort(host,port) }`
 is easier to read